examples/qml_features/rust/Cargo.toml

```diff
@@ -22,6 +22,7 @@ serde_json.workspace = true
 
 [build-dependencies]
 cxx-qt-build.workspace = true
+cxx-qt-lib.workspace = true
```

Comment:
this causes Windows builds to fail with a missing dll :thinking:

Reply:
(note it works when using cxx-qt-lib-headers, so is something specific to the build.rs and note it happened when using cxx_build instead of cxx-qt-build too)